plugins/Denizen/scripts/cooking/allergies.dsc

```diff
@@ -0,0 +1,78 @@
+allergies:
+    type: world
+    debug: false
+    events:
+        on player dies:
+
+            - define allergy <script[allergy_info].data_key[valid_allergies].keys.random>
+            - define effect <script[allergy_info].data_key[valid_effects].keys.random>
+            - flag player allergic:<[allergy]>
+            - narrate <[allergy]>
+            - flag player allergy_effect:<[effect]>
```

Comment:
Would probably be better to just use a map of allergy/effect. Is more compact and allows for multiple allergies

Reply:
? can you elaborate?

Reply:
rather than have 2 independent flags of allergy and allergy_effect, make the flag instead a map of allergy=alergy_effect. Since maps can hold many key/value pairs, this would allow one person to have multiple allergies at once (or multiple effects for one allergy)